Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -1,971 +1,953 @@
+                            : ""));
+
+            itContractRoles.forEach(role => {
+                const roleColumnId = `itContract${role.Id}`;
+                const roleKey = getRoleKey(role.Id);
+                const numberOfRolesToConcat = 5;
+                launcher = launcher
+                    .withColumn(builder =>
+                        builder
+                        .withDataSourceName(roleKey)
+                        .withTitle(role.Name)
+                        .withId(roleColumnId)
+                        .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Contains)
+                        .withoutSorting()
+                        .withContentOverflow()
+                        .withInitialVisibility(false)
+                        .withRendering(dataItem => Helpers.RenderFieldsHelper.renderInternalReference(
+                            `kendo-contract-${roleKey}-rendering`,
+                            "it-contract.edit.roles",
+                            dataItem.Id,
+                            Helpers.ArrayHelper.concatFirstNumberOfItemsAndAddElipsis(dataItem.roles[role.Id],
+                                numberOfRolesToConcat)))
+                        .withExcelOutput(
+                            dataItem => Helpers.ArrayHelper.concatFirstNumberOfItemsAndAddElipsis(
+                                dataItem.roles[role.Id],
+                                numberOfRolesToConcat)));
+            });
 
-                if (!idTofind) {
-                    // if no id was found then do nothing
-                    return;
-                }
+            launcher = launcher
+                .withColumn(builder =>
+                    builder
+                    .withDataSourceName("DataProcessingRegistrations")
+                    .withTitle("Databehandleraftale")
+                    .withId("dataProcessingRegistrations")
+                    .withoutSorting()
+                    .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.FixedValueRange)
+                    .withFixedValueRange(
+                        [
+                            Models.Api.Shared.YesNoIrrelevantOption.YES,
+                            Models.Api.Shared.YesNoIrrelevantOption.NO
+                        ].map(value => {
+                            return {
+                                textValue: Models.ViewModel.Shared.YesNoIrrelevantOptions.getText(value),
+                                remoteValue: value
+                            };
+                        }),
+                        false)
+                    .withContentOverflow()
+                    .withRendering(dataItem => {
+                        var activeDprs = [];
+                        dataItem.DataProcessingRegistrations.forEach(dpr => {
+                            //TODO: Comparing with an enum value always returns false
+                            if (dpr.IsAgreementConcluded.toString() === "YES") {
```

Comment:
I tried comparing the enum value with frontend enum or with int, but the only way I got it to work is when comparing the response value with 'YES', is there a way of solving this?